.github/workflows/ci.yml

```
@@ -0,0 +1,41 @@
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  go_version: '~1.20.12'
+
+jobs:
+  go_mod_tidy:
+    name: Check state of go.mod and go.sum
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)